Project.toml

```diff
@@ -7,6 +7,7 @@ version = "0.5.13"
 CategoricalArrays = "324d7699-5711-5eae-9e2f-1d82baa6b597"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+FilePaths = "8fc22ac5-c921-52a6-82fd-178b2807b824"
```

Comment:
Would `FilePathsBase` be sufficient? To avoid depending on MacroTools and URIParser